python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/__init__.py

```diff
@@ -64,6 +64,19 @@ def _uninstrument(self, **kwargs: Any) -> None:
     def get_span(self, run_id: UUID) -> Optional[Span]:
         return self._tracer.get_span(run_id) if self._tracer else None
 
+    def get_root_chain_spans(self, run_id: UUID) -> Optional[List[Span]]:
+        span = self.get_span(run_id)
+        root_chain_spans = []
+        while span and span.get_span_context().is_valid:
+            span_id = span.get_span_context().span_id
+            tracer = self._tracer
+            assert tracer
+            if span_id in tracer._root_span_ids:
+                root_chain_spans.append(span)
+
+            span = tracer._parent_span_by_span_id.get(span_id)  # get parent span
+        return root_chain_spans if root_chain_spans else None
```

Comment:
nit. might as well just return an empty list, since it makes no practical difference but simplifies the types.
```suggestion
        return root_chain_spans
```